subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r76/TestFailureProgressEventCrossVersionTest.groovy

```diff
@@ -276,6 +278,8 @@ class TestFailureProgressEventCrossVersionTest extends ToolingApiSpecification {
             dependencies {
                 testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.1'
                 testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.7.1'
+                testImplementation 'junit:junit:4.13.2'
```

Comment:
❔ Is this actually required?

Reply:
This was needed for the unused import only; removed.